docs/src/release-notes-js.md

```diff
@@ -6,6 +6,81 @@ toc_max_heading_level: 2
 
 import LiteYouTube from '@site/src/components/LiteYouTube';
 
+## Version 1.46
+
+### TLS Client Certificates
+
+TLS Client Authentication allows the server to request a client certificate and verify it.
+This release allows you to specify them for the browser and when using the [APIRequestContext] (API Testing).
+The following snippet will add a client certificate for `https://example.com` which gets used in the test runs:
+
+```ts
+import { defineConfig } from '@playwright/test';
+
+export default defineConfig({
+  // ...
+  use: {
+    clientCertificates: [{
+      origin: 'https://example.com',
+      certPath: './cert.pem',
+      keyPath: './key.pem',
+      passphrase: 'mysecretpassword',
+    }],
+  },
+  // ...
+});
+```
+
+Its also possible to manually specify them per-project ([project "use section"](./api/class-testproject#test-project-use)) or when creating a new context or APIRequestContext manually via [`method: Browser.newContext`] and [`method: APIRequest.newContext`].
+
+### Component Testing: New `router` fixture
+
+This release introduces a `router` fixture to intercept and handle network requests.
```

Comment:
This currently sounds like the release introduces the ability to intercept network requests. My understanding is that we're not adding an ability, but we're making it *easier* API-wise. Would this capture that better?
```suggestion
This release introduces a `router` fixture that's a new API to intercept and handle network requests.
```

Reply:
Only now did I realize that this is specific to component testing. How do you feel about this instead?

```diff
- This release introduces a `router` fixture to intercept and handle network requests.
+ This release introduces a `router` fixture to intercept and handle network requests in component tests.
```